reader.lua

```diff
@@ -97,7 +97,12 @@ local UIManager = require("ui/uimanager")
 local Device = require("device")
 local Font = require("ui/font")
 
--- read some global reader setting here:
+-- change some global default values according to the device types
+if Device:isKobo() then
+    KOBO_SCREEN_SAVER = "/mnt/onboard/screensaver"
```

Comment:
Can you put this into https://github.com/koreader/koreader/blob/master/frontend/device/kobo/device.lua#L81?